es6rest-persistence/src/main/java/com/netflix/conductor/dao/es6rest/dao/Elasticsearch6RestQueueDAO.java

```diff
@@ -488,7 +488,7 @@ public boolean wakeup(String queueName, String id) {
             Map<String, Object> map = new HashMap<>();
             map.put("popped", false);
             map.put("unacked", false);
-            map.put("deliverOn", System.currentTimeMillis());
+            map.put("deliverOn", 0L);
```

Comment:
deliverOn with 0 will go to the top during picking up the records. So by this change we making this update higher priority than any other existing records